test/sequential/test-util-debug.js

```diff
@@ -26,7 +26,7 @@ function test(environ, shouldWrite) {
 
   var spawn = require('child_process').spawn;
   var child = spawn(process.execPath, [__filename, 'child'], {
-    env: { NODE_DEBUG: environ }
+    env: { NODE_DEBUG: environ, HOME: process.env.HOME }
```

Comment:
@bnoordhuis I have an update on this. I talked to the maintainer of Lttng and they are now removing the library ability to print to stderr or stdout unless LTTNG-DEBUG env variable is active at compilation or at runtime. This will be implemented in lttng's next iteration, but until then I suggest we leave the test as is so it will pass.